pkg/bootstrap/docker/openshift/helper.go

```diff
@@ -161,10 +187,64 @@ func (h *Helper) OtherIPs(excludeIP string) ([]string, error) {
 	return resultIPs, nil
 }
 
+func KillExistingSocat() error {
+	_, err := os.Stat(SocatPidFile)
+	if os.IsNotExist(err) {
+		return nil
+	}
+	if err != nil {
+		return err
+	}
+	pidStr, err := ioutil.ReadFile(SocatPidFile)
+	if err != nil {
+		return err
+	}
+	pid, err := strconv.Atoi(string(pidStr))
+	if err != nil {
+		return err
+	}
+	process, err := os.FindProcess(pid)
```

Comment:
In this case, FindProcess will always return a process regardless if it exists or not. That's just the way the golang library behaves. I guess if an error is returned, something has gone really wrong.